python/pyspark/sql/dataframe.py

```diff
@@ -6503,13 +6503,13 @@ def toArrow(self) -> "pa.Table":
         Examples
         --------
         >>> df = spark.createDataFrame([(2, "Alice"), (5, "Bob")], schema=["age", "name"])
-        >>> df.toArrow()  # doctest: +SKIP
+        >>> df.coalesce(1).toArrow()
         pyarrow.Table
         age: int64
         name: string
         ----
-        age: [[2],[5]]
-        name: [["Alice"],["Bob"]]
+        age: [[2,5]]
+        name: [["Alice","Bob"]]
```

Comment:
```
        age: [[2],[5]]
        name: [["Alice"],["Bob"]]
```
vs
```
        age: [[2,5]]
        name: [["Alice","Bob"]]
```

just reflects the underlying `pyarrow.lib.ChunkedArray` which is affected by createDataFrame's partitioning (which is different between classic and connect), so add `coalesce(1)` to make this test always pass.